doc/source/examples/sharp-immersed-boundary-solver/sedimentation-64-particles/sedimentation-64-particles.rst

```diff
@@ -0,0 +1,349 @@
+
+Mesh
+~~~~~~
+.. code-block:: text
+
+    subsection mesh
+        set type                 = dealii
+    	set grid type            = subdivided_hyper_rectangle
+    	set grid arguments       = 1,1,12: 0,0,0 : 2 , 2 , 24 : true
+    	set initial refinement   = 4
+    end
+
+The domain is a rectangular box as such we can directly use a subdivided hyper rectangle mesh from the deal.II library. In this case, we have orientated the z-direction with gravity. As such, we have the long side of the box along this axis.
+
+* The ``grid arguments`` is set to  ``1,1,12: 0,0,0 : 2 , 2 , 24 : true``. This section has 3 subsections. First ``1,1,12`` describes the initial subdivision of the box. This subdivision has been chosen as it is the smallest mesh we can do of the box in order to have cubic elements. Secondly ``0,0,0 : 2 , 2 ,24`` describes the 2 points from which we have derived the rectangular box (0,0,0) and  (2,2,24). Finally, we have ``true``, which is a boolean to activate the coloration of the boundary. This allows us to define separate boundary conditions at each side of the box.
+
+* The ``initial refinement`` is set to 4. This will ensure to have a base mesh that is a bit smaller than the particle.
+
+
+Mesh adaptation control
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+.. code-block:: text
+
+    subsection mesh adaptation
+      # Fraction of coarsened elements
+      set fraction coarsening = 0.2
+    
+      # Fraction of refined elements
+      set fraction refinement = 0.025
+    
+      # How the fraction of refinement/coarsening are interepretedChoices are
+      # <number|fraction>.
+      set fraction type = number
+    
+      # Frequency of the mesh refinement
+      set frequency = 1
+    
+      # Maximum number of elements
+      set max number elements = 750000
+    
+      # Maximum refinement level
+      set max refinement level = 6
+      # minimum refinement level
+      set min refinement level = 4
+    
+      # Type of mesh adaptationChoices are <none|uniform|kelly>.
+      set type = kelly
+    
+      # Variable for kelly estimationChoices are <velocity|pressure>.
+      set variable = velocity
+    end
+
+* The ``fraction coarsening`` is set to 0.2. This limits the accumulation of elements when the particle is moving. It allows for cells far from the particle to be coarsened when the particles get further away.
+
+* The ``fraction refinement`` is set to 0.025. The objective here is to refine elements that become close to the particle when it's moving. This will mostly refine elements around the particle that are not included in the refinement zone around the particle. The refinement zone around the particle will be discussed in more detail in the IB particle section.
+
+* The ``set frequency`` is set to 1. Since the particle is moving at each time step, the refinement zone around it should be reevaluated at each time step.
```

Comment:
```suggestion
* The ``frequency`` is set to 1. Since the particle is moving at each time step, the refinement zone around it should be reevaluated at each time step.
```